javaparser-metamodel/src/test/java/com/github/javaparser/metamodel/PropertyMetaModelTest.java

```diff
@@ -0,0 +1,39 @@
+package com.github.javaparser.metamodel;
+
+import com.github.javaparser.ast.Modifier;
+import com.github.javaparser.ast.body.BodyDeclaration;
+import org.junit.Test;
+
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+
+public class PropertyMetaModelTest {
+    @Test
+    public void whenPropertyIsVerySimpleThenTypeInfoIsCorrect() {
+        PropertyMetaModel bert = new PropertyMetaModel(null, "bert", int.class, Optional.empty(), false, false, false, false);
+        assertEquals("int", bert.getTypeName());
+        assertEquals("int", bert.getTypeNameGenericsed());
```

Comment:
assertEquals("int", bert.getTypeNameGenericsed()); what does sed stand for? GenericSeed or GenericSet?